.github/workflows/compatibility.yml

```diff
@@ -0,0 +1,66 @@
+name: Integration for Compatibility
+
+on: [push, pull_request]
+
+jobs:
+  integration-test-compatibility:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        entry:
+          - { version: '1.1.0', secured: 'true' }
```

Comment:
May be Im totally off, I see we are replicating few pieces of code in all client repositories. 
Is there a way we can define these in a single plane (May be opensearch-clients) and consume it in all our client repos?

Reply:
@dblock Updated the workflow matrix.

Reply:
@saratvemulapalli Do you mean like having a template in `opensearch-clients`?

Reply:
I dont know what a template is :/. 
I was looking for a single place where common code can live. Lets take compatibility as an example:
- Defining this in a central place will make it easier when we add new version support.
- Enforces our CIs to test against new versions.
- Keeps all clients in sync.

Reply:
Oh got it! Yeah we are trying to keep it as consistent as possible across the client repos but I will keep this in mind if we can place common code. My current thought is having a single place might be difficult since the clients can be on different versions with different compatibility, their CIs are written differently etc. But will explore this once the breaking changes are done.